### PR TITLE
uv, devcontainer, jax

### PR DESCRIPTION
**Pull request by garibarba**

### CODE REVIEW

**Comment by garibarba** [comment]
Accuracy: 0.86
torch.cuda.is_available(): True

**Comment by garibarba** [comment]
Accuracy: 0.864
torch.cuda.is_available(): True
jax.default_backend(): gpu

**Comment by garibarba** [comment]
Accuracy: 0.868
torch.cuda.is_available(): True
jax.default_backend(): gpu
